packages/server/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "build": "rimraf dist/ && tsc -p tsconfig.build.json && mv dist/src/* dist/ && rimraf dist/src/ && yarn postbuild",
+    "debug": "yarn build && node --expose-gc --inspect=9222 dist/index.js",
```

Comment:
What relevance does the `expose-gc` flag have here? the only details I can find on it are that it will allow us to call `global.gc()` from within the program, is that something you did while attached in a debugger?

Reply:
yes, while I was debugging I was manually calling the `global.gc()` to make sure that the garbage collector had ran before I collected a snapshot, it just made sure that my testing of the solution was accurate.